web/themes/custom/accessibility_theme/src/patterns/components/hero/hero.twig

```diff
@@ -0,0 +1,53 @@
+{{ attach_library('accessibility_theme/hero') }}
+
+<div class="hero {{ modifier }}" {{ extra_attributes }}>
+  {#
+    Optional: Pass in Drupal specific functionality.
+    Note that this is only needed with nodes and blocks.
+    This isn't needed for Drupal Paragraphs.
+  #}
+  {#
+    {{ title_prefix }}
+    {{ title_suffix }}
+  #}
+
+  {# Media could be an image or video. #}
+  {% if image %}
+    <div class="hero__media">
+      {%
+        include '@accessibility_theme/media-item/media-item.twig' with {
+          "image": image
+        } only
+      %}
+    </div>
+  {% endif %}
+
+  <div class="hero__content">
+    {% if title %}
+      <div class="hero__heading">
```

Comment:
It looks like you are using the same css modifier twice.  You can remove this wrapper div completely and only leave the Heading on its own.